tpool/aio_linux.cc

```diff
@@ -174,11 +170,20 @@ class aio_linux final : public aio
 
   int bind(native_file_handle&) override { return 0; }
   int unbind(const native_file_handle&) override { return 0; }
+  const char *get_implementation() const override { return "Linux native AIO"; };
 };
 
 std::atomic<bool> aio_linux::shutdown_in_progress;
+}
+
+namespace tpool
+{
 
-aio *create_linux_aio(thread_pool *pool, int max_io)
+#ifdef HAVE_URING
+aio *create_libaio(thread_pool *pool, int max_io)
+#else
+aio *create_linux_aio(thread_pool *pool, int max_io, aio_implementation implementation __attribute__((unused)))
```

Comment:
Drop the `implementation __attribute__((unused))` from this line. Just `aio_implementation`. It's cleaner

It is C++, if you do not have to name parameter, if you do not use them.